quasar.conf.js

```diff
@@ -74,6 +74,10 @@ module.exports = function() {
           pathRewrite: {
             "^/api": ""
           }
+        },
+        "/methods": {
+          target: "http://localhost:3000",
+          changeOrigin: true
```

Comment:
What is this section intended to do? How does it align with the deployment?

E.g. in the deployment, anything prefixed with `/api` will get sent to the backend. Do we now need to maintain two separate sets of URL paths for development and deployment?

Reply:
I.e. why isn't the following configuration sufficient for local development?

https://github.com/GeriLife/wellbeing-client/blob/10921a4192586fb24a39e7645a236f4451483173/quasar.conf.js#L71-L75

Reply:
I would prefer to change the Axios request paths to align with the existing proxy configuration, rather than adding a second proxy configuration.

Reply:
In this deployment, http://95.217.223.95/#/, apis starting with `/methods` are failing are they aren't hitting the right route
![Screenshot from 2021-04-12 20-50-15](https://user-images.githubusercontent.com/10625985/114419308-f1f80c00-9bd0-11eb-8aa5-f7988205231c.png)

Reply:
Please add the ‘/api/’ prefix to those requests rather than a new proxy configuration.

Reply:
For clarity, any request to the backend should be prefixed with “/api/”. The prefixed request will be provided to the backend and the prefix will be removed. This way the development and deployment environments share a common proxy configuration. The proxy configuration has already been merged so no new proxy configuration is necessary. Please update the services files to align with the existing proxy configuration and remove the additional proxy configuration from this and any other pull request.